fdbserver/DataDistribution.actor.cpp

```diff
@@ -4794,7 +4895,7 @@ ACTOR Future<Void> storageServerTracker(
 				TraceEvent(SevInfo, "InvalidWiggleServer", self->distributorId)
 				    .detail("Address", worstAddr.toString())
 				    .detail("ProcessId", server->lastKnownInterface.locality.processId())
-				    .detail("ValidWigglingId", self->wigglingPid.present());
+				    .detail("ValidWigglingId", self->wigglingId.present());
```

Comment:
nit: "ValidWigglingId" ; why do we need to have "Valid" as prefix; do we record "Invalid" ids too ? :-)